app/src/org/commcare/android/view/GraphView.java

```diff
@@ -405,6 +405,7 @@ private void configureSeries(SeriesData s, XYSeriesRenderer currentRenderer) {
             currentRenderer.setPointStyle(style);
             currentRenderer.setFillPoints(true);
             currentRenderer.setPointStrokeWidth(2);
+            mRenderer.setPointSize(6);

```

Comment:
I wonder if it would be a good idea to try to scale this relative to the view's resolution.